### PR TITLE
English readme

### PR DESCRIPTION
**Pull request by Joeper214**
Please Review and Merge PR. @pocke. :bow:

### CODE REVIEW

**Comment by pocke** [comment]
I'll review it

**Comment by pocke** [comment]
but I'm not good at English ... :scream_cat:

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # SkyHopper
 システム自動構築ツール
 
+## Language
```
Comment (pocke) [inline README.md]: I think `## language` header is not needed. 

e.g.
https://github.com/rakuten-nlp/rakutenma/blob/master/README-ja.md

Comment (Joeper214) [inline README.md]: :+1:

File: doc/installation/en/skyhopper.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
+# SkyHopper deployment procedure
+
+By using [SkyHopper Cookbooks](https://github.com/skyarch-networks/skyhopper_cookbooks/tree/master/cookbooks/skyhopper), you can automate your system by installing a package.
+
+If you used cookbook, we will proceed to [Creating MySQL user](#user-content-creating-mysq-user).
```
Comment (pocke) [inline doc/installation/en/skyhopper.md]: typo anchor (`mysq-user`)

**Comment by pocke** [comment]
LGTM thanks!!

**Pull request merged**